drivers/SmartThings/zigbee-water-leak-sensor/fingerprints.yml

```
@@ -99,3 +99,8 @@ zigbeeManufacturer:
     manufacturer: THIRDREALITY
     model: 3RWS18BZ
     deviceProfileName: water-battery
+  - id: "Sengled E1L-G7K"
+    deviceLabel: Sengled Water Leak Sensor
+    manufacturer: sengled.
```

I believe the manufacturer name should just be "sengled"
```suggestion
    manufacturer: sengled
```